main.py

```diff
@@ -144,10 +147,40 @@ def check_date(store, date_to_check):
         '{}: {} -> {}'.format(op, last[astart:aend], booked_list[bstart:bend])
         for op, astart, aend, bstart, bend in  difflib.SequenceMatcher(None, last, booked_list).get_opcodes()
         if op != 'equal'
-    ]
+    ] #Contains the difference between the current details and the previous history file.
 
     if diff:
         pprint(diff)
+        send_email(diff)
+
+def send_email(diff):
```

Comment:
with all the environment variable comments below, they should come from send_email arguments:

```python
def send(email_host, email_port, user, password, from, to):
```

Then when calling send, fill the arguments from environment variables. 

Passing arguments is better than accessing os.environ which is essentially a global. Only use os.environ contained in one spot, then pass around as arguments

Reply:
Also with these arguments send_email becomes more useful and able to be used on multiple servers and send to different recipients, amongst other things.